src/gpu_macros.h

```diff
@@ -176,13 +122,25 @@
 #define gpuFloatComplex  hipFloatComplex
 #define make_gpuDoubleComplex make_hipDoubleComplex
 #define make_gpuFloatComplex make_hipFloatComplex
-
 #endif
 #define gpuCheckLastError(...) GPU_CHECK_ERROR(gpuGetLastError())
 #else
-// bool gpu_support() { return false;}
-// inline int num_available_gpus(){ return 0; } 
+inline int gpu_support() { return 0;}
```

Comment:
Wait, wasn't this defined already earlier?

Reply:
They were commented out. I brought them back with adjustments for the current code.